Fluid/Ast/IdentifierSegment.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
 using Fluid.Values;
 
 namespace Fluid.Ast
 {
     public class IdentifierSegment : MemberSegment
     {
-        private IMemberAccessor _accessor;
+        private readonly ConcurrentDictionary<Type, IMemberAccessor> _accessors = new ConcurrentDictionary<Type, IMemberAccessor>();
```

Comment:
It's not worth it, just replace the `_accessor` cache by a local variable and don't cache it.
Or, only reuse the cache if the _accessor and another (new) _type property are the same. Most times the model sent to a template is the same, so this will work in most cases. Just be sure to "set" the _accessor after the _type if you test if _accessor is set first.

Reply:
If I use layout and cache, then I have to accommodate multiple different models in the layout.

Reply:
> It's not worth it, just replace the _accessor cache by a local variable and don't cache it.

Oops seems I did think about the lastes changes I made, it was fine .. I will update the PR soon